### PR TITLE
Internal: Remove un-accessible markup from nested tabs [ED-10751]

### PR DESCRIPTION
**Pull request by rami-elementor**

### CODE REVIEW

File: modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js [hunk 1]
```diff
@@ -236,13 +235,6 @@ export default class NestedTabs extends Base {
 		elementorFrontend.elements.$window.on( 'elementor/nested-tabs/activate', this.reInitSwipers );
 	}
 
-	preventDefaultLinkBehaviourForTabTitle( event ) {
```
Comment (hein-obox) [inline modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js:239]: Have you checked if this function is used in the Mega Menu?

In the Mega Menu with need 'a href' links.

**Comment by hein-obox** [comment]
Is there a Jira task for this with expected outcomes?
In our team we usually create a mindmap to map all possible scenarios.

@davseve What do you think?

**Comment by hein-obox** [comment]
@rami-elementor I am away for a couple of days, therefore I asked @davseve to have a look.

**Comment by hein-obox** [comment]
@rami-elementor Have you checked why the Playwright tests are failing?

File: modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js (hunk 1 above)
Comment (rami-elementor) [reply modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js:239]: Yes, I checked, the function is not not in use.

**Review by hein-obox — DISMISSED** [review]

**Review by arielk — DISMISSED** [review]

**Review by TzviRabinovitch — APPROVED** [review]

**Pull request merged**